values/jobs/copy-certs.gotmpl

```diff
@@ -0,0 +1,9 @@
+{{- $v := .Values }}
+{{- $teams := keys $v.teamConfig.teams }}
+
+type: CronJob
+env:
+  TEAM_IDS: '{{ $teams | sortAlpha | toJson }}'
```

Comment:
I don't know if you have the same problem, but when I hit template with a non-empty array of teams I get the following configmap:

```yaml
  otomi:template:log # Source: jobs/templates/configmap.yaml
  otomi:template:log apiVersion: v1
  otomi:template:log kind: ConfigMap
  otomi:template:log metadata:
  otomi:template:log   name: job-copy-certs-env
  otomi:template:log   labels:
  otomi:template:log     helm.sh/chart: jobs-0.1.0
  otomi:template:log     app.kubernetes.io/name: jobs
  otomi:template:log     app.kubernetes.io/instance: job-copy-certs
  otomi:template:log     app.kubernetes.io/version: "0.1.0"
  otomi:template:log     app.kubernetes.io/managed-by: Helm
  otomi:template:log data:
  otomi:template:log   TEAM_IDS: "[]"
  ```

  Can you reproduce it? I think this is supposed to be non-empty right?

Reply:
Empty arrays are like that

Reply:
I just copied the default one in the tests/fixtures/.secrets to my ENV_DIR

Reply:
I think you did not do it right as I can't reproduce. Please use the values from `tests/fixtures/.env`.

Reply:
Cool, yes I just wanted to make sure that we didn't miss anything.